docs/source/model_design/general_reference_material/bliss/race_ethnicity/index.rst

```diff
@@ -100,10 +100,137 @@ audience.
 1.1 Research question
 ---------------------
 
-.. todo:: 
+When you’re designing a research question, it’s important to consider both if race, 
+ethnicity, and/or ancestry are relevant, and, if so, what the casual pathways might be. Spending 
+time considering the causal pathways will help you consider what exactly you are 
+investigating and might identify more relevant proxy measures.
+
+To work through this example, let’s consider a starting research question: how will 
```

Comment:
I like that CVD is our recurring example throughout this section, it provides clarity/consistency. It could also be helpful to offer an example research question where race/ethnicity/ancestry would not be as relevant to a research question?